client/index.html

```diff
@@ -261,14 +267,14 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
-									<% themes.forEach(function(themeName) { %>
-										<option value="<%= themeName %>">
-											<%= themeName.charAt(0).toUpperCase() + themeName.slice(1) %>
+									{{#each themes}}
+										<option value="{{this}}">
+											{{firstLetterUppercase this}}
```

Comment:
I don't care re: naming. I'm not native speaker so I would probably look at stuff like https://atom.io/packages/change-case or the like, see what people do.

Re: structure of theme, I don't think this PR should care about that. My opinion is to just port it in this PR then deal with themes in a further PR. It seems out-of-place and an easy source of bikeshedding while this PR is mostly ready. Just my 2c.